benches/cases/rewriting.rs

```diff
@@ -13,7 +13,7 @@ define_group!(
 
                     Ok(())
                 })],
-                ..Settings::default()
+                ..Settings::new()
```

Comment:
Why changing this to `new`?

Reply:
`Settings::new()` creates settings for a non-`Send` rewriter (that is, it creates a `Settings<'_, '_, HandlerNormalTypes>`).

`Settings::new_send()` creates settings for a `Send` rewriter (that is, it creates a `Settings<'_, '_, HandlerSendTypes>`).

`Settings::default()` also creates settings for a non-`Send` rewriter, just like `Settings::new()`.

The reason I added `Settings::new()` is because I had to add `Settings::new_send()` so it would make sense to also have `Settings::new()` as well.